main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+        for player_name, player_info in data.items():
```

Comment:
Do not overload context manager

Reply:
I meant this https://github.com/mate-academy/py-game-models/blob/master/checklist.md#1-do-not-overload-the-context-manager